Handdeck/Include/Handdeck.hpp

```diff
@@ -0,0 +1,51 @@
+#pragma once
+#include <vector>
+#include <string>
+#include <iostream>
+#include <string>
+#include <vector>
+#include <cstdlib>
+#include <ctime>
+
+
+using Card = std::string;
+
+class Deck
+{
+public:
+  std::vector<Card> Cards;
+  void addCard(Card cardToAdd);
+};
+
+class DiscardDeck :public Deck
+{
+public:
+  int maxSize;
+  DiscardDeck(int _maxSize);
+};
+
+class PullDeck :public Deck
+{
+public:
+  int maxSize;
+  PullDeck(int _maxSize);
+};
+
+class TableDeck :public Deck

```

Comment:
co to jest? co to reprezentuje?

Reply:
Wywaliliśmy, trzeba będzie go lepiej zaimplementować.